src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of class or method implementations. Please remove the empty line.